src/Symfony/Component/Process/Tests/AbstractProcessTest.php

```diff
@@ -564,28 +564,6 @@ public function testIsNotSuccessful()
         $this->assertFalse($process->isSuccessful());
     }
 
-    public function testProcessIsNotSignaled()
-    {
-        if ('\\' === DIRECTORY_SEPARATOR) {
-            $this->markTestSkipped('Windows does not support POSIX signals');
-        }
-
-        $process = $this->getProcess(self::$phpBin.' -v');
-        $process->run();
-        $this->assertFalse($process->hasBeenSignaled());
-    }
-
-    public function testProcessWithoutTermSignalIsNotSignaled()
```

Comment:
they are not, because of overwrites changing the behavior of some tests based on sigchild being enabled (see child classes). ``hasBeenSignaled`` and ``getTermSignal`` need to be tested separately to handle these cases properly.